.github/workflows/subctl-unit.yml

```
@@ -0,0 +1,40 @@
+---
+name: Downstream Compatibility Check
```

```suggestion
name: Consuming Projects
```

IMHO it would be more consistent with similar testing in Shipyard.
Also it's shorter so it renders better on GHA (all other workflow names are rather short)

WDYT?

---

That’s a great idea, I like the consistency!